exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapRecordReader.java

```diff
@@ -136,7 +136,7 @@ public void close() throws Exception {
   }
 
   private ImmutableList<ProjectedColumnInfo> getProjectedColsIfItNull() {
-    return projectedCols != null ? projectedCols : initCols(new Schema());
+    return projectedCols != null ? projectedCols : initCols(new Schema(false));
```

Comment:
Please add comment explaining why it is always false here.

Reply:
I don't know.  I'm going to deprecate this class because it actually is no longer needed.

Reply:
Actually, never mind. I'm deleting this class, because it was replaced by the `PcapBatchReader` when I updated this to use EVF.